docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,67 @@
 # Jira integration
 
-Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+Pulse integrates with Jira Cloud to receive:
+
+-   Data about **incidents**, necessary to calculate the following Accelerate metrics:
+
+    -   [Time to recover](../metrics/accelerate.md#time-to-recover)
+
+    -   [Change failure rate](../metrics/accelerate.md#change-failure-rate)
+
+-   Data bout **issues**, necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
 
 ## Setting up the Jira integration
 
+!!! note
+    -   The Jira integration must be set up by a **Jira administrator**, as Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    -   You can only integrate one Pulse organization with each Jira instance.
+    -   [See below](#jira-permissions) the permissions that Pulse requires from your Jira instance.
+
 To set up the Jira integration:
 
 1.  On Pulse, [expand **Integrations** and select **Jira**](https://app.pulse.codacy.com/integrations/jira){: target="_blank"}.
 
-1.  Click **Install Jira** and follow the instructions on the Jira UI to install the app.
+1.  Click **Install Jira App** and follow the instructions on the Jira UI to install the app. You must be a Jira administrator to proceed with this step.
 
-    !!! important
-        This step should be performed by a Jira administrator, since Pulse will only have access to Jira resources that the user setting up the integration has access to.
+    ![Installing the Pulse Jira app](images/jira-installing.png)
 
-        You can only integrate one Pulse organization with each Jira instance.
+1.  Confirm that Pulse successfully created the integration and the necessary webhooks on Jira. If there was an error please [contact support](mailto:pulsesupport@codacy.com).
```

Comment:
The webhook creation part is only done when 'Complete setup' is clicked. Maybe we can just remove that reference since users can't really check if webhooks were installed.